com.woltlab.wcf/templates/userCard.tpl

```diff
@@ -7,15 +7,41 @@
 				alt="">
 		</div>
 		<div class="userCard__header__avatar">
-			{user object=$user type='avatar64' ariaHidden='true' tabindex='-1'}
+			{unsafe:$user->getAvatar()->getImageTag(64)}
 
 			{if $user->isOnline()}<span class="userCard__onlineIndicator jsTooltip" title="{lang username=$user->username}wcf.user.online.title{/lang}"></span>{/if}
 		</div>
+
+		<div class="userCard__header__interactions">
+			{if $__wcf->user->userID && $user->userID != $__wcf->user->userID}
+				{if !$__wcf->getUserProfileHandler()->isIgnoredByUser($user->userID)}
+					{if $__wcf->getUserProfileHandler()->isFollowing($user->userID)}
```

Comment:
I am aware that this is most likely copied from somewhere else but maybe introduce two helper methods instead?